components/deleteConfirmation.js

```diff
@@ -8,9 +8,20 @@ const deleteConfirmation = (message, id) => {
   const popupWrapper = document.createElement("div");
   popupWrapper.className = "popup-wrapper";
 
+  const popupLogo = document.createElement("div");
+  popupLogo.className = "popup-logo";
+  popupLogo.innerHTML = "<img src='./public/icon-delete.svg' alt='icon' />";
+
   const popup = document.createElement("div");
   popup.className = "popup";
-  popup.innerText = message;
+
+  const popupText = document.createElement("div");
+  popupText.innerText = message;
+
+  setTimeout(() => {
+    popup.className += " animation";
+    popupWrapper.className += " animation";
+  }, 1);
```

Comment:
An alternative way that is less bug prone:

```suggestion
  setTimeout(() => {
    popup.classList.add("animation")
    popupWrapper.classList.add("animation")
  }, 1);
```

Notice that we don't have to worry about adding a space in front of `className`.